src/index.html

```diff
@@ -1,13 +1,54 @@
 <!doctype html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
-        content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <title>Product cards</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <h1>Product cards</h1>
-</body>
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport"
+          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <title>Product cards</title>
+    <link rel="stylesheet" href="./styles/main.scss">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Estonia&family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
+  </head>
+  <body class="body">
+
+    <div class="Card" data-qa="card">
+      <img
+        class="Card-Image"
+        src="images/imac.jpeg"
+        alt="Product image">
+
+      <div class="Card-ProductData">
+        <p class="Card-DeviceName">APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)</p>
+        <p class="Card-DeviceCode">Product code: 195434</p>
+      </div>
```

Comment:
Use BEM classnaming. Fix all cases
`block__element--modifier`
```suggestion
      <div class="card__product-data">
        <p class="card__product-name">APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)</p>
        <p class="card__product-code">Product code: 195434</p>
      </div>
```

Reply:
![image](https://github.com/mate-academy/layout_product-cards/assets/68586602/eff49c57-046c-4492-863a-9d69576c7060)

Reply:
Чому не можу використати React style??

Reply:
kebab-case є стандартним підходом до найменування класів у HTML та CSS. Використання kebab-case в BEM дозволяє зберегти однорідність і узгодженість з наявним HTML та CSS кодом, що полегшує розуміння та обслуговування проекту
2. Сумісність з іншими інструментами та середовищами: Багато інструментів, таких як редактори коду, автоматизаційні системи, транспілятори CSS і т.д., підтримують kebab-case як стандартний формат для назв класів. Це полегшує роботу з проектом та забезпечує більшу сумісність з екосистемою інструментів.
3. Читабельність: Kebab-case назви класів можуть бути легше читати та розуміти, особливо при роботі з довгими назвами класів або в складних структурах BEM

+react style, як не дивно прийнято використовувати вже у react проектах
<img width="1023" alt="image" src="https://github.com/mate-academy/layout_product-cards/assets/70220667/6b64d17e-ee6b-4411-8827-a961becd6d37">